drivers/at86rf215/at86rf215_ofdm.c

```diff
@@ -0,0 +1,325 @@
+    }
+
+    return 0;
+}
+
+/* Table 6-90. Recommended Transmitter Frontend Configuration */
+static uint32_t _TXCUTC_LPFCUT(uint8_t option)
+{
+    switch (option) {
+        case 1: return 10 << TXCUTC_LPFCUT_SHIFT;
+        case 2: return 8 << TXCUTC_LPFCUT_SHIFT;
+        case 3: return 5 << TXCUTC_LPFCUT_SHIFT;
+        case 4: return 3 << TXCUTC_LPFCUT_SHIFT;
+    }
+
+    return 0;
+}
+
+/* Table 6-90. Recommended Transmitter Frontend Configuration */
+static uint32_t _TXDFE_SR(uint8_t option)
+{
+    switch (option) {
+        case 1:
+        case 2: return 3 << TXDFE_SR_SHIFT;
+        case 3:
+        case 4: return 6 << TXDFE_SR_SHIFT;
+    }
+
+    return 0;
+}
+
+/* Table 6-90. Recommended Transmitter Frontend Configuration */
+static uint32_t _TXDFE_RCUT(uint8_t option)
+{
+    switch (option) {
+        case 1: return 3 << TXDFE_RCUT_SHIFT;
+        case 2:
+        case 3: return 3 << TXDFE_RCUT_SHIFT;
+        case 4: return 2 << TXDFE_RCUT_SHIFT;
+    }
+
+    return 0;
+}
+
+/* Table 6-93. Recommended PHY Receiver and Digital Frontend Configuration */
+static uint32_t _RXDFE_RCUT(uint8_t option, bool superGHz)
+{
+    switch (option) {
+        case 1: return 4 << RXDFE_RCUT_SHIFT;
+        case 2: return 2 << RXDFE_RCUT_SHIFT;
+        case 3: return (2 + superGHz) << RXDFE_RCUT_SHIFT;
+        case 4: return 1 << RXDFE_RCUT_SHIFT;
+    }
+
+    return 0;
+}
+
+/* Table 6-93. Recommended PHY Receiver and Digital Frontend Configuration */
+static uint32_t _RXBWC_BW(uint8_t option, bool superGHz)
+{
+    switch (option) {
+        case 1: return (9 + superGHz) << RXBWC_BW_SHIFT;
+        case 2: return 7 << RXBWC_BW_SHIFT;
+        case 3: return (4 + superGHz) << RXBWC_BW_SHIFT;
+        case 4: return (2 + superGHz) << RXBWC_BW_SHIFT;
+    }
+
+    return 0;
+}
+
+/* Table 6-93. Recommended PHY Receiver and Digital Frontend Configuration */
+static uint32_t _RXBWC_IFS(uint8_t option, bool superGHz)
+{
+    switch (option) {
+        case 1:
+        case 2: return 1;
+        case 3: return superGHz;
+        case 4: return !superGHz;
+    }
+
+    return 0;
+}
+
+static void _set_option(at86rf215_t *dev, uint8_t option)
+{
+    const bool superGHz = !is_subGHz(dev);
+
+    /* Set Receiver Bandwidth */
+    at86rf215_reg_write(dev, dev->RF->RG_RXBWC,
+                        _RXBWC_BW(option, superGHz) | _RXBWC_IFS(option, superGHz));
+    /* Set fS (same as TX); fCUT for RX */
+    at86rf215_reg_write(dev, dev->RF->RG_RXDFE,
+                        _TXDFE_SR(option) | _RXDFE_RCUT(option, superGHz));
+    /* Set Power Amplifier Ramp Time; fLPCUT */
+    at86rf215_reg_write(dev, dev->RF->RG_TXCUTC,
+                        RF_PARAMP8U | _TXCUTC_LPFCUT(option));
+    /* Set fS; fCUT for TX */
+    at86rf215_reg_write(dev, dev->RF->RG_TXDFE,
+                        _TXDFE_SR(option) | _TXDFE_RCUT(option));
+
+    /* set channel spacing */
+    at86rf215_reg_write(dev, dev->RF->RG_CS, _channel_spacing_kHz(option) / 25);
```

Comment:
Just noticed this 25, why is it?

Reply:
Oh yeah 25Khz resolution

Reply:
Can you add a comment like for `o-qpsk`?

`/* set channel spacing with 25 kHz resolution */`

Reply:
Done! And I also undid some of the damage done by `uncrustify` in the next line.